pnpm-workspace.yaml

```
@@ -1,3 +1,5 @@
 packages:
   - 'js/*'
-  - 'client/js'
+  - "gradio"
+  - "client/*"
+  - "client/*/*"
```

This isn't supposed to be
```suggestion
  - "client/**/*"
```
is it?

---

I might actually just make this client/python/*. The reason I didn't use the more general glob is because I don't want it to make stuff inside node modules which I think the looser glob would. Will check.